### ISSUE TITLE
Application Error (0xc0000005)

### ISSUE THREAD

**Issue by davidcardenasus** [standardnotes/forum#520]
After installing Standard Notes, I get this error whenever I attempt to start the program. 

"The application was unable to start correctly (0xc0000005). Click OK to close the application."

This occurred about a month ago as well. Reinstalled Windows 10 and I receive the same error. I am thinking this has something to do with one of my settings in Windows Security? More specifically, one of the exploit protection settings, but I am not sure.

**Comment by mobitar** [comment]
Hmm..haven't seen this one before. A quick search reveals that this seems to be a common issue and might have something to do with picky antivirus software or registry settings. Have you tried some of the various troubleshooting tips suggested online?

**Comment by davidcardenasus** [comment]
> 
> 
> Hmm..haven't seen this one before. A quick search reveals that this seems to be a common issue and might have something to do with picky antivirus software or registry settings. Have you tried some of the various troubleshooting tips suggested online?

Figured it out! Enabling "Force randomization for images (Mandatory ASLR)" setting in Windows Defender under Exploit Protection will cause this issue. Its turned off by default, but I had it turned on. After disabling this setting and restarting my computer, I was able to run Standard Notes.
![SN](https://user-images.githubusercontent.com/47157243/56048807-c7e88680-5d37-11e9-8032-48910303e69f.png)
